README.md

```diff
@@ -8,8 +8,8 @@ More technically, it will find all 'Pathway' instances with a 'doi' attribute th
 from the instance's 'stableIdentifier' attribute. 
 
 New features and tests have been implemented for this iteration of UpdateDOIs: 
-  - Runtime tests ensure conchordance of the instance's being updated in the Test Reactome and GK Central databases
-  - Users can provide a report file (UpdateDOIs.report) of DOIs that are expected to be updated. It can be created from the data found <a href="https://docs.google.com/spreadsheets/d/1KtZ_Z3rvBELroubmeO1ai5otbsS26QpXZn6au-oSCWw/edit#gid=1011530219">here</a>. This is explained further in <a href="https://github.com/reactome/data-release-pipeline/new/develop/update-dois#updatedoisreport">UpdateDOIs.report</a> section.
+  - Runtime tests ensure concordance of the instance's being updated in the Test Reactome and GK Central databases
+  - Users can provide a report file (UpdateDOIs.report) of DOIs that are expected to be updated. It can be created from the data found <a href="https://docs.google.com/spreadsheets/d/1KtZ_Z3rvBELroubmeO1ai5otbsS26QpXZn6au-oSCWw/edit#gid=1011530219">here</a>. This is explained further in <a href="#updatedoisreport">UpdateDOIs.report</a> section.
```

Comment:
This README is now pretty much entirely out of date since I had to make some pretty significant structural changes to make it work with Jenkins. The end result is a better module, but the description is not up to date. For example, if no report file is provided the step will now produce its own report file without updating the DB.